auto3dseg/tasks/hecktor22/README.md

```diff
@@ -0,0 +1,57 @@
+
+# HECKTOR22
+
+
+The HECKTOR22 dataset is from MICCAI 2022 challenge **[HEad and NeCK TumOR Segmentation and Outcome Prediction (HECKTOR22)](https://hecktor.grand-challenge.org)**. The solution described here won the 1st place in the HECKTOR22 challenge [(NVAUTO team)](https://hecktor.grand-challenge.org/final-leaderboard/):
```

Comment:
If data access needs sign-ups and permission, it would be nice to mention the availability. Thanks!

Reply:
ok